cluster-scope/overlays/ocp-staging/oauths/cluster_patch.yaml

```diff
@@ -24,6 +24,8 @@ spec:
       name: moc-testing
       openID:
         claims:
+          id:
```

Comment:
I don't see `id` as a property for `.spec.identityProviders[].openID.claims` in the document here: https://docs.openshift.com/container-platform/4.9/rest_api/config_apis/oauth-config-openshift-io-v1.html

Reply:
Ah, you're right. I got mislead by outdated documentation targeting 3.11 :) 

I achieved the same result by creating a claim called `sub` and adding that to the client on Keycloak.